bower.json

```diff
@@ -1,27 +1,31 @@
 {
   "name": "gaia-switch",
-  "version": "0.0.0",
-  "main": "gaia-switch.js",
+  "version": "0.2.0",
   "homepage": "https://github.com/gaia-components/gaia-switch",
   "authors": [
     "Wilson Page <wilsonpage@me.com>"
   ],
+  "main": "gaia-switch.js",
   "license": "MIT",
   "ignore": [
     "**/.*",
+    "!.jshintrc",
```

Comment:
I think we should deprecate this pattern. It was originally added to stop gaia linting bower_components with its own rules. But we should be .`jshintignore`ing our third-party stuff any way.